bin/ChangeLog

```diff
@@ -0,0 +1,2612 @@
+2017-06-08 Andrea Bocci <andrea.bocci@cern.ch>
```

Comment:
why do you copy this from the root directory ?

Reply:
Mistake I guess. i will delete it from the bin. Might have to do something with Eclipse.

Reply:
Still not deleted ?